docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+# Command-Line Tutorial
+
+This tutorial will explain how to use the HELM command line tools to run benchmarks, aggregate statistics, and visualize results.
+
+In this tutorial, we will run two runs using the `mmlu` scenario on the `huggingface/gpt-2` model. The `mmlu` scenario implements the **Massive Multitask Language (MMLU)** benchmark from [this paper](https://arxiv.org/pdf/2009.03300.pdf), and consists of a Question Answering (QA) task using a dataset with questions from 57 subjects such as elementary mathematics, US history, computer science, law, and more. We will run two runs: the first using questions about anatomy, and the second using questions about philosophy.
+
+## Using `helm-run`
+
+`helm-run` is a command line tool for running benchmarks.
+
+To run this benchmark using the HELM command-line tools, we need to specify **run spec descriptions** that describes the desired runs. For this example, the run spec descriptions are `mmlu:subject=anatomy,model=huggingface/gpt-2` (for anatomy) and `mmlu:subject=philosophy,model=huggingface/gpt-2` (for philosophy).
+
+Next, we need to create a **run spec configuration file** contining these run spec descriptions. A run spec configuration file is a text file containing `RunEntries` serialized to JSON, where each entry in `RunEntries` contains a run spec description. The `description` field of each entry should be a **run spec description**. Create a text file named `run_spec.conf` with the following contents:
+
+```
+entries: [
+  {description: "mmlu:subject=anatomy,model=huggingface_gpt-2", priority: 1},
+  {description: "mmlu:subject=philosophy,model=huggingface_gpt-2", priority: 1},
+]
+```
+
+We will now use `helm-run` to execute the runs that have been specified in this run spec configuration file. Run this command:
+
+```
+helm-run --run-specs run_spec.conf --local --suite v1 --max-eval-instances 10
+```
+
+The meaning of the additional arguments are as follows:
+
+- `--local` specifies not to use the Stanford CRFM API. You should always use `--local` unless you are a Stanford CRFM API user.
```

Comment:
I wonder if we should make `--local` the default (when no proxy server is explicitly provided)?

Reply:
Filed #1202 for making --local default.